lib/Wheel.js

```diff
@@ -1,6 +1,12 @@
 class Wheel {
   constructor() {
-    
+  this.currentElement = null;
+}
```

Comment:
Indentation is off here, and a handful of other spots. Make sure to run your code through a linter to clean these things up.